webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -360,7 +360,7 @@ class Boot extends Loggable {
     // Do nothing instead, as at allows to keep open tabs context until we get the new cookie
     // This does not affect security as it is only a redirection anyway and did not change
     // the session itself.
-    LiftRules.noCometSessionCmd.default.set(() => JsCmd.unitToJsCmd(()))
+    LiftRules.noCometSessionCmd.default.set(() => JsRaw(s"createErrorNotification('You have been signed out. Please reload the page to sign in again.')").cmd)
```

Comment:
We might check in the future if we want to do the same thing for `noAjaxSessionCmd`